CODEOWNERS

```diff
@@ -14,9 +14,12 @@ azure-pipelines.yml @garyverhaegen-da
 azure-cron.yml      @garyverhaegen-da
 
 # Language
-/compiler/          @hurryabit
-/libs-haskell/      @hurryabit
-/daml-assistant/    @associahedron
+/compiler/          @hurryabit @cocreature
```

Comment:
Do I really have to be a codeowner? I very much like to be able to differentiate between people requesting my review explicitly which I get as email notifications and just codeowner notifications which I mostly get simply by looking at all PRs.